### ISSUE TITLE
Error In Readme #2

### ISSUE THREAD

**Issue by aerickso** [jopo666/HistoEncoder#2]
Hi

Thanks again for Histoencoder. Another minor issue with the readme. (Conda environment, Python 3.9, Windows 11 Home, 64bit).

```
HistoEncoder extract --input my/local/directory/tile_images --model-name prostate-small

Error: No such option: --input (Possible options: --input-size, --input_dir)
```

Needed to change both --input to --input_dir and prostate-small to prostate_small

```
HistoEncoder extract --input_dir "my/local/directory/tile_images" --model-name prostate_small
```

**Comment by aerickso** [comment]
Same --input --> --input_dir for next line.

**Comment by jopo666** [comment]
Fixed by b4bc1dd
